docs/manual.md

```diff
@@ -258,10 +278,15 @@ INSTANCE y2k WITH BIRTH_YEAR <- 80, LICENSE_AGE <- 18
 ```
 
 The downside of this approach is that you have to declare the variables of the
-extended specification.
+extended specification. This is easy with only two variables, but can quickly
+become unwieldy.
 
 ## 5.2. Convention over configuration
 
+Alternatively, you can extend the base module and ...
+<!-- TODO(shonfeder): how does the following work exactly? Nothing obvious comes up -->
```

Comment:
@konnov or @andrey-kuprianov for my own edification, I'm hoping to get a bit of clarification on this point.

Reply:
There is a special convention, that if smth with OVERRIDE_<x> prefix is found, it overrides the <x> declaration. Just a hack, actually, but quite easy to use in some circumstances

Reply:
Ah, I figured! So this is a hole in my TLA+ knowledge :) I looked for this in the TLA manual but didn't turn anything up. Thanks for explaining!

Reply:
It is not a standard TLA+ construct, but a convenient syntactic convention in Apalache. Nobody but me seems to like it though :-)

Reply:
Ah ha! We may want to document this a bit more clearly then, unless it's covered somewhere else. I'll make an issue to do that in a followup. Thanks!